consensus/broadcast/delayedBroadcast.go

```diff
@@ -606,6 +607,16 @@ func (dbb *delayedBlockBroadcaster) registerInterceptorsCallbackForShard(
 	return nil
 }
 
+func (dbb *delayedBlockBroadcaster) shardIdentifiers() map[uint32]struct{} {
```

Comment:
This should have been part of the shardcoordinator implementation. Maybe we will do it one day.